crates/bevy_sprite/src/sprite.rs

```diff
@@ -16,4 +16,46 @@ pub struct Sprite {
     /// An optional custom size for the sprite that will be used when rendering, instead of the size
     /// of the sprite's image
     pub custom_size: Option<Vec2>,
+    /// [`Anchor`] point of the sprite in the world
+    pub anchor: Anchor,
+}
+
+/// How a sprite is positionned relative to it's [`Transform`](bevy_transform::components::Transform).
+/// It defaults to `Anchor::Center`.
+#[derive(Debug, Clone, Reflect)]
+pub enum Anchor {
+    Center,
+    BottomLeft,
+    BottomCenter,
+    BottomRight,
+    CenterLeft,
+    CenterRight,
+    TopLeft,
+    TopCenter,
+    TopRight,
+    /// Custom anchor point. Top Left is `(-0.5, 0.5)`.
```

Comment:
This should be a bit more explicit that the value is scaled to the sprite's scale.